aesara/tensor/subtensor_opt.py

```diff
@@ -1666,3 +1668,112 @@ def local_subtensor_SpecifyShape_lift(fgraph, node):
         new_shape_arg = as_tensor(shape_arg[indices], dtype=np.int64, ndim=1)
 
     return [specify_shape(new_obj_arg, new_shape_arg)]
+
+
+def is_same_var_or_constant(var1, var2):
```

Comment:
If you need to check that two term graphs are (syntactically) the same, use the function in `aesara.graph.basic`.  That will cover more cases, as well.

Reply:
Thanks. I assume you mean `equal_computations`